torchrl/modules/utils/mappings.py

```diff
@@ -65,7 +65,11 @@ def mappings(key: str) -> Callable:
 
     Args:
         key (str): one of "softplus", "exp", "relu", "expln",
-        or "biased_softplus".
+        or "biased_softplus". If key beggins with "biased_softplus",
+        then the key needs to take the following form:
+        "biased_softplus_{bias}" where bias is a float and the bias term of the biased softplus.
```

Comment:
`"biased_softplus_{bias}"` (in code style)
floating point number

where `"bias"` can be converted to a floating point number that will be used to bias the softplus function.